ui/web/index.html

```diff
@@ -167,11 +181,12 @@ <h1>Work Estimation</h1>
 
         <div id="place">
 
-            <input type="submit" name="but" id="loadTasksButton" value="Load">
+            <input type="submit" name="but" id="loadTasksButton" value="Load" onmousedown="openRoadMap()">
```

Comment:
Road map should appear after tasks are loaded but not only after user moves mouse to to "Load" button and then out of it.

Reply:
Changes are made to the index.html file